### PR TITLE
Add deprecated notice in readme

### PR DESCRIPTION
**Pull request by joshuali925**
Signed-off-by: Joshua Li <joshuali925@gmail.com>

### Description
Add deprecated notice in readme
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+__NOTE:__ We have merged this plugin into [Observability Plugin](https://github.com/opensearch-project/trace-analytics/) since OpenSearch 1.2, please use notebooks in Observability Dashboards plugin. Thanks.
```
Comment (davidcui1225) [inline README.md:1]: np- the "Thanks" appended at the end seems unnecessary/almost passive-aggressive?

**Review by davidcui1225 — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (joshuali925) [reply README.md:1]: oh, removing it...

**Review by davidcui1225 — DISMISSED** [review]
LGTM- looks like common-utils failed to build in the CI though?

**Comment by joshuali925** [comment]
> LGTM- looks like common-utils failed to build in the CI though?

probably because we didn't move to maven central, but this repo is deprecated and this might be the last PR so i don't think it's necessary to fix

**Comment by davidcui1225** [comment]
> > LGTM- looks like common-utils failed to build in the CI though?
> 
> probably because we didn't move to maven central, but this repo is deprecated and this might be the last PR so i don't think it's necessary to fix

Sounds good- although with the new repo rules I can't remember if PRs can be merged in with a failing CI

**Review by zhongnansu — APPROVED** [review]

**Review by ps48 — APPROVED** [review]

**Pull request merged**